src/index.html

```diff
@@ -4,24 +4,400 @@
       rel="stylesheet"
-      href="styles/index.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="header__link"
+      >
+        <img
+          class="header__logo"
+          src="/src/images/logo.png"
+          alt="Logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#gadgets"
+              class="nav__link"
+            >
+              GADGETS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#tablets"
+              class="nav__link"
+            >
+              TABLETS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              PHOTO
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#video"
+              class="nav__link"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="container">
+        <div class="card">
+          <img
+            class="card__photo"
+            src="./images/imac.jpeg"
+            alt="laptop"
+          />
+          <div class="card__text">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__code">Product code: 195434</div>
+          <div class="stars stars--1">
+            <div class="star"></div>
+            <div class="star"></div>
+            <div class="star"></div>
+            <div class="star"></div>
+            <div class="star"></div>
+            <span class="review">Reviews: 5</span>
+          </div>
+          <div class="price">
+            <div class="price__item">Price:</div>
+            <div class="price__item2">$2,199</div>
+          </div>
```

Comment:
Fix classes and tag everywhere
```suggestion
          <div class="card__price">
            <p class="card__price--label">Price:</p>
            <p class="card__price--value">$2,199</p>
          </div>
```